### ISSUE TITLE
Oppertunity

### ISSUE THREAD

**Issue by Anticope12** [AsadMir10/Crypto-bot-using-MACD-indicator#1]
Hi @AsadMir10 ,
I have noticed you are working on Mind2Web project. 

I just want us to connect for a project opportunity, how can I contact you?

**Comment by AsadMir10** [comment]
Yeah Sure, This is my Linkden Handle, you can connect here, https://www.linkedin.com/in/asad-ahsan-4258a5201/
